### PR TITLE
chore: remove specs not used by rules

### PR DESCRIPTION
**Pull request by xiangce**
- see RHINRULE-335

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

**Comment by xiangce** [comment]
@JoySnow - This PR is consistent with the update to the uploader.json, please kindly review.

**Review by JoySnow — APPROVED** [review]
@xiangce , I am good with the changes.

File: insights/specs/__init__.py [hunk 1]
```diff
@@ -321,7 +320,6 @@ class Specs(SpecSet):
     kexec_crash_size = RegistryPoint()
     keystone_conf = RegistryPoint()
     keystone_crontab = RegistryPoint()
-    keystone_crontab_container = RegistryPoint()
```
[the remark this answers is not in the document]
Comment (xiangce) [reply insights/specs/__init__.py:324]: @JoySnow - removed. and I also removed some other unused RegistryPoint, please check again. Thanks.

**Review by JoySnow — APPROVED** [review]

File: insights/specs/__init__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoySnow) [reply insights/specs/__init__.py:324]: Thanks @xiangce . And I am good with the new changes.

**Comment by xiangce** [comment]
All IQE tests are passed except for RHEL 7&6 which are failed due to ENV issues

**Pull request merged**